src/wmnds/patterns/question/reference/_reference.njk

```diff
@@ -0,0 +1,47 @@
+{% macro wmndsReference(params) %}
+{# Imports #}
+{% from "wmnds/components/form-elements/radios/_radios.njk" import wmndsRadios %}
+{% from "wmnds/components/form-elements/question/_question.njk" import wmndsQuestion %}
+{% from "wmnds/components/form-elements/progress-indicator/_progress-indicator.njk" import wmndsProgressIndicator %}
+{% from "wmnds/components/link/link/_link.njk" import wmndsLink %}
+
+{# Set vars and defaults #}
+{% set question = params.question if params.question else "How would you...?" %}
+{% set text = params.text if params.text %}
+{% set fields = params.fields if params.fields %}
+{% set actualSection = params.actualSection if params.actualSection else "1" %}
+{% set totalSections = params.totalSections if params.totalSections else "X" %}
+{% set sectionName = params.sectionName if params.sectionName else "Section Name" %}
+
+{{
+    wmndsLink( { isBackLink: true, text: "Back" } ) | trim | indent(4)
+}}
+
+<div class="bg-white wmnds-m-t-lg">
+    <div class="wmnds-col-1 wmnds-col-sm-3-4 wmnds-col-lg-2-3  wmnds-p-sm  wmnds-p-b-none">
+        {{
+            wmndsProgressIndicator({
+                actualSection: actualSection,
+                totalSections: totalSections,
+                sectionName: sectionName
+            }) | trim | indent(4)
+        }}
+        {{
+            wmndsQuestion({
+                text: question
+            }) | trim | indent(4)
+        }}
+        {{ text | safe | trim | indent(4)}}
+
+        <div class="wmnds-p-t-md">
+            {{ fields  | safe | trim | indent(4) }}
+        </div>
+
+    </div>
+    <div class="wmnds-col-1  wmnds-p-sm">
+        {{ params.button  | safe  | trim | indent(4) }}
+    </div>
+
+</div>
+
+{% endmacro %}
```

Comment:
Check your indenting again.
Also, you are missing the form and fieldset elements like your "form element" pattern.